File: Getting Started/PythagoreanTriplet.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int a,b,c;
    cin >> a >> b >> c;
    if ((a * a == b * b + c * c) || (b * b == a * a + c * c) || (c * c == a * a + b * b)) {
        cout << "true";
    }
    else{
        cout << "false";
    }
    return 0;
    //write your code here
    
}

File: Getting Started/PythagoreanTriplet.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        // write your code here  
        Scanner scn = new Scanner(System.in);
        int a = scn.nextInt();
        int b = scn.nextInt();
        int c = scn.nextInt();
        if ((a * a == b * b + c * c) || (b * b == a * a + c * c) || (c * c == a * a + b * b)) {
            System.out.println("true");
        } else {
            System.out.println("false");
        }

    }
}